### PR TITLE
chore: add volunteer manthanank

### PR DESCRIPTION
**Pull request by manthanank**

### CODE REVIEW

File: data/users/Manthanank.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      "slug": "programming",
+      "topics": ["JavaScript"]
+    },
+    {
+      "slug": "software-development",
+      "topics": ["MEAN"]
+    },
+    {
+      "slug": "web-development",
+      "topics": ["HTML", "CSS", "Angular", "Nodejs", "Expressjs"]
```
Comment (codewithvoid) [inline data/users/Manthanank.json:16]: restrict this to max 2-3 topics

File: data/users/Manthanank.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "bio": "Front End Developer",
+  "image": "https://avatars.githubusercontent.com/u/50883320?s=40&v=4",
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": ["JavaScript"]
+    },
+    {
+      "slug": "software-development",
+      "topics": ["MEAN"]
```
Comment (codewithvoid) [inline data/users/Manthanank.json:12]: MEAN should be part of web-development

File: data/users/Manthanank.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Manthan Ank",
+  "bio": "Front End Developer",
+  "image": "https://avatars.githubusercontent.com/u/50883320?s=40&v=4",
```
Comment (codewithvoid) [inline data/users/Manthanank.json:4]: use this url for image https://avatars.githubusercontent.com/u/50883320?v=4

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks @manthanank for volunteering. Can you please make requested changes.

**Comment by manthanank** [comment]
Requested changes made

**Review by codewithvoid — APPROVED** [review]
Thanks for making the changes quickly. Welcome to the community.

**Pull request merged**